### DISCUSSION TITLE
Permanent error 500 on one issue page

### DISCUSSION THREAD

**Discussion by Glandos** [community/community#23940] [Projects and Issues]
<p>I’ve tried to reach <a href="https://github.com/vuejs/vetur/issues/1187" class="inline-onebox" rel="noopener nofollow ugc">SFC without a script tag show an error when trying to import · Issue #1187 · vuejs/vetur · GitHub</a> from different browser, emptied profiles, etc.</p>
<p>But still, it is always an error 500 from the server. Is there something to do?</p>

**Comment by canuckjacq** [comment]
<p>Hey <a class="mention" href="/glandos">@glandos</a> that looks like a :bug: - I’ve raised an internal issue and I’ll do my best to keep you updated.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:tada: this should be fixed for you now <a class="mention" href="/glandos">@glandos</a></p>
